tests/sentry/workflow_engine/handlers/condition/test_event_frequency_handlers.py

```diff
@@ -0,0 +1,80 @@
+import pytest
+
+from sentry.models.group import Group
+from sentry.rules.conditions.event_frequency import EventFrequencyCondition
+from sentry.testutils.skips import requires_snuba
+from sentry.utils import json
+from sentry.workflow_engine.handlers.condition.event_frequency_handlers import (
+    EventFrequencyConditionHandler,
+)
+from sentry.workflow_engine.models.data_condition import Condition
+from tests.sentry.workflow_engine.handlers.condition.test_base import (
+    ConditionTestCase,
+    EventFrequencyQueryTestBase,
+)
+
+pytestmark = [pytest.mark.sentry_metrics, requires_snuba]
+
+
+class TestEventFrequencyCondition(ConditionTestCase):
+    condition = Condition.EVENT_FREQUENCY
+    rule_cls = EventFrequencyCondition
+    payload = {
+        "interval": "1h",
+        "id": "sentry.rules.conditions.event_frequency.EventFrequencyCondition",
+        "value": 1000,
+    }
+
+    def test(self):
+        dc = self.create_data_condition(
+            type=self.condition,
+            comparison=json.dumps({"interval": "1h", "value": 1000}),
+            condition_result=True,
+        )
+
+        self.assert_passes(dc, 1000)
+
+        self.assert_does_not_pass(dc, 999)
+
+    def test_dual_write(self):
+        dcg = self.create_data_condition_group()
+        dc = self.translate_to_data_condition(self.payload, dcg)
+
+        assert dc.type == self.condition
+        assert dc.comparison == json.dumps({"interval": "1h", "value": 1000})
+        assert dc.condition_result is True
+        assert dc.condition_group == dcg
+
+
+class EventFrequencyQueryTest(EventFrequencyQueryTestBase):
```

Comment:
this is the same as https://github.com/getsentry/sentry/blob/e85b3f9735f6594284000fa81b23cbeca6bbe88c/tests/snuba/rules/conditions/test_event_frequency.py#L133